ansible/roles/schulcloud-calendar/templates/configmap.yml.j2

```
@@ -9,7 +9,7 @@ data:
   NODE_ENV: "production"
   CORS_ORIGIN: "http://calendar-svc:3000"
   # Paths
-  SCHULCLOUD_BASE_PATH: "http://api-svc:3030"
+  SCHULCLOUD_BASE_PATH: "http://api-svc:3030/api/v1"
   NOTIFICATION_SERVICE_BASE_PATH: "http://notification-svc"
   SCHULCLOUD_ALL_USERS_FOR_UUID: "http://api-svc:3030/resolve/users/"
   SCHULCLOUD_ALL_SCOPES_FOR_TOKEN: "http://api-svc:3030/resolve/scopes/"
```

What is her with api?
```suggestion
  SCHULCLOUD_ALL_USERS_FOR_UUID: "http://api-svc:3030/api/v1/resolve/users/"
  SCHULCLOUD_ALL_SCOPES_FOR_TOKEN: "http://api-svc:3030/api/v1/resolve/scopes/"
```